subsys/usb/class/netusb/function_rndis.c

```diff
@@ -222,6 +218,8 @@ static struct __rndis {
 static u8_t manufacturer[] = CONFIG_USB_DEVICE_MANUFACTURER;
 static u32_t drv_version = 1U;
 
+static u8_t tx_buf[1500];
+
```

Comment:
This array holds the Ethernet packet, but also the RNDIS header. I guess its size should therefore be `1500 + sizeof(struct rndis_payload_packet)` or even better `NETUSB_MTU + sizeof(struct rndis_payload_packet).

Reply:
Fixed